remo/base/static/base/css/app-fd4.less

```diff
@@ -2135,32 +2134,42 @@ meter::-o-meter-suboptimum-value {
         .copyblock {
             padding-bottom: 10px;
         }
-        .required-field {
-            top: -30px;
-            margin-bottom: -30px;
-        }
         button {
             margin-top: 10px;
         }
     }
     .category-select {
         margin-top: 10px;
-        .required-field {
-            top: 10px;
-        }
-    }
-    .map-point-select {
-        button + .required-field {
-            top: 10px;
-        }
     }
+    
     #event-attend-button button {
         padding: 8px 10px 10px;
     }
     .event-single-title {
         font-weight: bold;
         margin-bottom: 30px;
     }
+
+    .section {
+        padding-right: 25px;
+        padding-bottom: 25px;
+    }
+
+    .required-field {
+        &:extend(.required-field);
+        left: 5px;
+        top:0;
```

Comment:
if it's to be vertical aligned to the middle, do you mean

.required-field-input top: 20px
.required-field top: 18px

_top: 20px_
![input-20](https://cloud.githubusercontent.com/assets/3946307/24780367/4c2bf39a-1b69-11e7-9c20-ddb8ad5bbd2d.png)

_top: 10px_
![input-10](https://cloud.githubusercontent.com/assets/3946307/24780385/6c60a110-1b69-11e7-9778-8312ba75a0f8.png)